css-layout-classes/index.html

```diff
@@ -8,49 +8,46 @@
 
   <body>
 
-    <div id="body">
-      <div id="title">
+    <div class="container">
+      <div class="col-full">
         <h1>Choose your character!</h1>
       </div>
 
 
       <div class="row">
 
-        <div class="container" id="container-one">
 
-          <div id="image">
-            <img src="images/link.png" alt="link from zelda">
-          </div>
+        <div class="col-third" id="row-one">
```

Comment:
Remove this id. This `div` is a column, not a row, so this is just adding confusion